packages/keystone/src/fields/types/text/index.ts

```diff
@@ -70,6 +70,7 @@ export const text =
       },
     };
 
+    // defaulted to false as a zero length string is preferred
```

Comment:
```suggestion
    // defaulted to false as a zero length string is preferred to null
```

Reply:
// defaulted to false as a zero length string is preferred to null
```suggestion
    // defaulted to false as a zero length string is preferred
```